MapboxCoreNavigationTests/CocoaPodsTest/PodInstall/Podfile.lock

```diff
@@ -1,16 +1,16 @@
 PODS:
   - Mapbox-iOS-SDK (5.5.0)
-  - MapboxCoreNavigation (0.37.0):
+  - MapboxCoreNavigation (0.37.1):
```

Comment:
Not sure why the previous version should be used here.

Reply:
The CocoaPods installation test lags official releases by design. We haven’t released v0.37.2 yet, so the test can’t verify that running `pod update` would install v0.37.2. However, it would detect if a newer dependency would be pulled in. This is why the test has failed quite often during the v1.0 development process, as breaking changes were introduced between prereleases of MapboxDirections.